transport_nantes/topicblog/views.py

```diff
@@ -572,6 +572,11 @@ def get_context_data(self, **kwargs):
             raise ImproperlyConfigured("base_model must be a class with a "
                                        "send_object_url attribute")
         context["send_to_view"] = self.base_model.send_object_url
+        context["base_model"] = self.base_model
+        context["sent_object"] = self.base_model.objects.filter(
```

Comment:
Not hugely important, but isn't this the object we are _about to send_, not something _already sent_?

Reply:
It is the _object about to be sent_